### PR TITLE
Julevask 2023

### PR DESCRIPTION
**Pull request by jonathan-eq**
Resolves https://github.com/equinor/scout/issues/966

### CODE REVIEW

**Review by kwinkunks — COMMENTED** [review]
Some suggestions:

- Update the versioning in GitHub Actions, eg `actions/checkout@v3` should be `v4` and `python-setup` should be `v5`. There may be others.
- Remove Packages and maybe Releases as well from the main page (in About cog thingy)
- Remove Hacktoberfest tag and maybe add some relevant ones like 'Enterprise', 'Distribution' sort of thing
- We are publishing the docs to GitHub Page in one of the actions, but they are not deployed. I think this just needs to be enabled in Settings > Pages. They should appear here https://equinor.github.io/komodo (there's a link in the README but it 404s)

**Comment by jonathan-eq** [comment]
Great feedback @kwinkunks! I will update it right away.
The docs are published in a github actions step, but it does not seem to work. See issue #423

File: README.md [hunk 1]
```diff
@@ -81,28 +81,35 @@ To use this environment, type `source builds/stable-0.0.1/enable`.
...
-- `komodo-transpiler` &mdash; Build release files
+- `komodo-post-messages` &mdash; Post messages to a release
+- `komodo-reverse-deps` &mdash; Extracts dependencies from a given set of
+packages
 - `komodo-show-version` &mdash; Return the version of a specified package in the active release
+- `komodo-snyk-test` &mdash; Test a release for security and license issues
+- `komodo-suggest-symlinks` &mdash; Returns a pull request if the symlink
+configuration could be updated
+- `komodo-transpiler` &mdash; Build release files
+
```
Comment (berland) [inline README.md:108]: added blank lines?

Comment (jonathan-eq) [reply README.md:108]: It got left behind after I sorted the commands😅
Removed them now!

**Review by xjules — APPROVED** [review]
I think it looks very good now @jonathan-eq !

**Pull request merged**